ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -747,3 +747,9 @@ def remainder(self, y, name=None):
 
     def is_floating_point(self):
         return paddle_frontend.is_floating_point(self._ivy_array)
+
+    @with_supported_dtypes(
+        {"2.5.1 and below": ("float32", "float64", "int32", "int64")}, "paddle")
+
+    def backward(self,x,grad_tensor=None,retain_graph=False,):
```

Comment:
Also, you will need to add a test for the function you implemented. Feel free to read through our [docs](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html#ivy-frontend-tests) and use the rest of the existing tests as reference.